kore/test/Test/Kore/Step/SMT/Representation/Builders.hs

```diff
@@ -189,7 +190,8 @@ unresolvedSmthookSymbol encodedName resultSort argumentSorts =
         , declaration = AST.SymbolBuiltin
             AST.IndirectSymbolDeclaration
                 { name = AST.AlreadyEncoded (AST.Atom encodedName)
-                , resultSort = AST.SortReference resultSort
-                , argumentSorts = map AST.SortReference argumentSorts
+                , sortDependencies =
```

Comment:
For my refactoring, I'm thinking about making `unresolvedSmthookSymbol` take all `Kore.Sort` arguments in one list, so it would have an argument `sortDependencies` instead of `resultSort` and `argumentSorts`